samza-core/src/main/java/org/apache/samza/coordinator/leaderelection/LeaderElector.java

```diff
@@ -28,7 +30,7 @@
    *
    * @return True, if caller is chosen as a leader through the leader election process. False, otherwise.
    */
-  boolean tryBecomeLeader();
+  boolean tryBecomeLeader(LeaderElectorListener leaderElectorListener);
 
   /**
```

Comment:
I am not sure if we need this method in general. I would rather remove it for now.